src/configure/configurers/provisioningConfigurer.ts

```diff
@@ -288,4 +293,13 @@ export class ProvisioningConfigurer implements IProvisioningConfigurer {
             files,
         } as DraftPipelineConfiguration;
     }
+
+    private async moveWorkflowFilesToRepoLocally(): Promise<void> {
+        const directoryPath: string = await this.localGitRepoHelper.getGitRootDirectory();
```

Comment:
rename to `gitRootDirectory`. usually a very common name like `directoryPath` does not convey precise info to reader

Reply:
done